nio_cli/commands/new.py

```diff
@@ -10,7 +10,18 @@ def __init__(self, options, *args, **kwargs):
         self._name = self.options['<project-name>']
 
     def run(self):
-        clone = "git clone git@github.com:{}/{}.git {}"
-        subprocess.call(
-            clone.format('nioinnovation', 'project_template', self._name),
-            shell=True)
+        clone = (
+            "git clone --depth=1 git@github.com:{}/{}.git {}"
+        ).format('nioinnovation', 'project_template', self._name)
+        submodule_update = (
+            'cd ./{} '
+            '&& git submodule update --init --recursive'
+        ).format(self._name)
+        reinit_repo = (
+            'cd ./{} '
```

Comment:
Those `git` commands need to be done from within the git repo. So after cloning it down, you need to change directories into it.